tests/test_miniworld.py

```diff
@@ -51,6 +51,20 @@ def test_pytorch_wrapper():
     env.close()
 
 
+def test_stochastic_wrapper():
```

Comment:
This test technically doesn't actual test that the wrapper other than the wrapper can be instantiated and `reset` and `step` work